pkg/overlay/cache.go

```diff
@@ -408,3 +408,21 @@ func getIP(ctx context.Context, target string) (_ string, err error) {
 	}
 	return ipAddr.String(), nil
 }
+
+func getNetwork(ctx context.Context, target string) (network string, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	ip, err := getIP(ctx, target)
+	if err != nil {
+		return "", err
+	}
+
+	addr := net.ParseIP(ip)
+	if addr == nil {
+		return "", errors.New("invalid ip")
+	}
+
+	//Filter all IP Addresses into /24 Subnet's
+	mask := net.CIDRMask(24, 32)
```

Comment:
We should check first if `len(addr) == 4`. If `len(addr) == 16` instead, we should not apply a mask (as discussed, there doesn’t seem to be any very reasonable way to guess the appropriate subnet size with ipv6 addresses, so we will punt on masking those for now)

Reply:
Ill add the check and the migration 👍

Reply:
For the migration, `UPDATE nodes SET last_ip = host(network(set_masklen(last_ip::INET, 24))) WHERE last_ip <> '' AND family(last_ip::INET) = 4;` ought to make the necessary changes

Reply:
I included a grouping per /64 subnet. I know that is not the best way, but better than allowing it entirely there..